airbyte-integrations/java-template-destination/readme.md

```diff
@@ -0,0 +1,20 @@
+### How to write a destination integration in java.
```

Comment:
there are holes in this readme that we can't fill until some of the documentation comes together. filled in what i can so that it is clear what we do know. otherwise placeholders.